src/providers/MeatInfoProvider.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState, createContext, useEffect } from 'react'
+
+
+export const MeatInfoContext = createContext();
+
+
+export const MeatInfoProvider = (props) => {
+    const [selectedMeatItemInfoFromSearchBar, setSelectedMeatItemInfoFromSeachBar] = useState('');
+    const [selectedAddOns, setSelectedAddOns] = useState([{ name: null, price: 0 }]);
+    const [isRemoveButtonOnDom, setIsRemoveButtonOnDom] = useState(false);
```

Comment:
We talked about moving this into a display provider.
Why did you create duplicate states?